tests/test_apps.py

```diff
@@ -266,11 +266,11 @@ def test_app_exclusion_filter(self):
         self.assertEqual(self.ctrl.state.state, "ready")
 
         # App was added
-        self.assertIn("appA-1", self.ctrl.state["rezContexts"])
-        context_a = self.ctrl.state["rezContexts"]["appA-1"]
+        self.assertIn("app_A-1", self.ctrl.state["rezContexts"])
+        context_a = self.ctrl.state["rezContexts"]["app_A-1"]
```

Comment:
I think the key (`app_request`) should be `"app_A==1.0.0"`

Reply:
Looks like that was it, thanks! (I'm a bit new to the allzpark internals, obviously lol)